mlos_bench/db/db_schema_sqlite3_duckdb.sql

```diff
@@ -0,0 +1,83 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- SQLite3/DuckDB version of the DB schema.
+
+DROP TABLE IF EXISTS trial_telemetry;
+DROP TABLE IF EXISTS trial_results;
+DROP TABLE IF EXISTS trial_config;
+DROP TABLE IF EXISTS experiment_merge;
+DROP TABLE IF EXISTS trial_status;
+DROP TABLE IF EXISTS experiment_config;
+
+CREATE TABLE experiment_config (
+    exp_id VARCHAR(255) NOT NULL,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
+    PRIMARY KEY (exp_id)
+);
+
+CREATE TABLE trial_status (
+    exp_id VARCHAR(255) NOT NULL,
+    trial_id INTEGER NOT NULL,
+    ts_start TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    ts_end TIMESTAMP,
+    -- Should match the text IDs of `mlos_bench.environment.Status` enum:
+    status VARCHAR(16),
```

Comment:
I guess there's no enums in DuckDB/Sqlite?
Maybe add a constraint instead to match the behavior of the postgres schema?
One other option could be to store the `int` value of the status that's in the code and then provide a view for mapping the int to the string description.  I think this is what SqlServer does in a lot of it's system tables for instance.

Reply:
one portable solution would be to have a const table with all possible status values and have an FK to it

Reply:
I'll add this to the schema now. I think we can even keep using string IDs - storing ints won't save us much space or cpu cycles, but would require an extra join for every user query

Reply:
nah, never mind - changing foreign keys on the fly is not as easy as I had thought before.. I'll roll back the change. BTW DuckDB does support enums - maybe we can implement it later if we decide to have different dialects of the same schema again. Let's discuss on Monday